build/images/ovs/Dockerfile.windows

```diff
@@ -0,0 +1,68 @@
+# Copyright 2022 Antrea Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+FROM mcr.microsoft.com/windows/servercore:ltsc2019 as downloader
+ADD https://aka.ms/vs/17/release/vc_redist.x64.exe /
+
+FROM mcr.microsoft.com/windows/servercore:ltsc2019
+# Set working directory
+WORKDIR c:\\
+
+# Switch to a privileged user
+USER ContainerAdministrator
+
+COPY --from=downloader /vc_redist.x64.exe /
+SHELL ["powershell", "-Command", "$ErrorActionPreference = 'Stop'; $ProgressPreference = 'SilentlyContinue';"]
+
+# Run the installation in PowerShell
+
+RUN powershell -Command "Start-Process -FilePath C:\vc_redist.x64.exe -ArgumentList ‘/install’,’/quiet’,’/norestart’ -Verb RunAs -Wait"
+
+# Download and extract Open vSwitch
+ADD https://downloads.antrea.io/ovs/ovs-2.15.2-antrea.0-win64.zip .
+RUN powershell -Command Expand-Archive ovs-2.15.2-antrea.0-win64.zip -DestinationPath .
```

Comment:
It's not suggested way to add a zip file and keep it in the image.
Please check this best practice:  https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#add-or-copy

Reply:
we need to clean zip file